File: averagecollector/src/main/java/com/chrisrecinos/view/controller/StartController.java
package com.chrisrecinos.view.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

/**
 * @author - Christopher Recinos
 */

@Controller
@RequestMapping(value = "/")
public class StartController {

    @RequestMapping(method = RequestMethod.GET)
    String getStartPage() {
        return "index";
    }
}

File: mynba2kdb/src/main/kotlin/com/chrisrecinos/mynba2kdb/controller/StartController.kt
package com.chrisrecinos.mynba2kdb.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RequestMethod

/**
 * @author - Christopher Recinos
 */

@Controller
@RequestMapping(value = ["/"])
class StartController {

    @RequestMapping(method = [RequestMethod.GET])
    fun getIntroPage() : String {
        return "index"
    }
}